text/maps/Route22.asm

```diff
@@ -1,104 +1,101 @@
 _Route22RivalBeforeBattleText1::
-	text "<RIVAL>: Hey!"
+	text "<RIVAL>: היי!"
 	line "<PLAYER>!"
 
-	para "You're going to"
-	line "#MON LEAGUE?"
+	para "אתה הולך לליגת"
+	line "ה#ימון?"
 
-	para "Forget it! You"
-	line "probably don't"
-	cont "have any BADGEs!"
+	para "אין סיכוי!"
+	line "אין מצב שיש לך"
+	cont "מספיק תגים!"
 
-	para "The guard won't"
-	line "let you through!"
+	para "השומר לא יתן"
+	line "לך להכנס!"
 
-	para "By the way, did"
-	line "your #MON"
-	cont "get any stronger?"
+	para "דרך אגב,"
+	line "ה#ימונים שלך"
+	cont "כבר חזקים יותר?"
 	done
 
 _Route22RivalAfterBattleText1::
-	text "I heard #MON"
-	line "LEAGUE has many"
-	cont "tough trainers!"
+	text "שמעתי שבליגת"
+	line "ה#ימון יש הרבה"
+	cont "מאמנים קשוחים!"
 
-	para "I have to figure"
-	line "out how to get"
-	cont "past them!"
+	para "אני חייב למצוא דרך"
+	line "להתגבר עליהם!"
```

Comment:
נראה לי שהכוונה היא יותר... חרטטנית? לא יודעת בדיוק איך לנסח את זה. כאילו אם הוא פשוט רצה לשתף איתך שהוא רוצה להיות חזק יותר, הוא היה אומר "I have to figure out how to beat them!".
אבל במקום, הוא אומר "get past them".
הם יותר כמו מכשול מאשר אתגר בשבילו. הוא לא מתעניין באמת בקרב, הוא רק רוצה לעקוף. אלה הבדלים דקים אבל מרגיש שהם עושים את כל העניין :)
יש לי הצעות, אם אתה אוהב מוזמן לקחת, אם יש לך הצעה חזקה יותר אשמח לשמוע.
```suggestion
	para "אני חייב למצוא"
	line "דרך לעבור אותם!"
```
```suggestion
	para "אני חייב להבין איך"
	line "לעבור אותם!"
```